packages/babel-parser/test/fixtures/es2015/uncategorised/240/input.js

```diff
@@ -1 +0,0 @@
-((a)) => 42
```

Comment:
This test is removed because it is almost identical to https://github.com/babel/babel/blob/32279147e6a69411035dd6c43dc819d668c74466/packages/babel-parser/test/fixtures/es2015/arrow-functions/inner-parens/input.js#L1